docker-compose/mysql/scripts/setup-apim4.0.0-IS5.11.0.sql

```diff
@@ -10,753 +10,749 @@ openbank_openbankingdb
+    VALIDITY_PERIOD BIGINT,
+    STATE VARCHAR (25) DEFAULT 'ACTIVE',
+    TOKEN_ID VARCHAR(255),
+    SUBJECT_IDENTIFIER VARCHAR(255),
+    PKCE_CODE_CHALLENGE VARCHAR(255),
+    PKCE_CODE_CHALLENGE_METHOD VARCHAR(128),
+    AUTHORIZATION_CODE_HASH VARCHAR(512),
+    IDP_ID INTEGER DEFAULT -1 NOT NULL,
+    PRIMARY KEY (CODE_ID),
+    FOREIGN KEY (CONSUMER_KEY_ID) REFERENCES IDN_OAUTH_CONSUMER_APPS(ID) ON DELETE CASCADE
+    )ENGINE INNODB;
 
 
 CREATE TABLE IF NOT EXISTS IDN_OAUTH2_AUTHZ_CODE_SCOPE(
-            CODE_ID   VARCHAR(255),
-            SCOPE     VARCHAR(60),
-            TENANT_ID INTEGER DEFAULT -1,
-            PRIMARY KEY (CODE_ID, SCOPE),
-            FOREIGN KEY (CODE_ID) REFERENCES IDN_OAUTH2_AUTHORIZATION_CODE (CODE_ID) ON DELETE CASCADE
-)ENGINE INNODB;
+    CODE_ID   VARCHAR(255),
+    SCOPE     VARCHAR(60),
+    TENANT_ID INTEGER DEFAULT -1,
+    PRIMARY KEY (CODE_ID, SCOPE),
+    FOREIGN KEY (CODE_ID) REFERENCES IDN_OAUTH2_AUTHORIZATION_CODE (CODE_ID) ON DELETE CASCADE
+    )ENGINE INNODB;
 
 CREATE TABLE IF NOT EXISTS IDN_OAUTH2_DEVICE_FLOW (
-            CODE_ID VARCHAR(255),
-            DEVICE_CODE VARCHAR(255),
-            USER_CODE VARCHAR(25),
-            CONSUMER_KEY_ID INTEGER,
-            LAST_POLL_TIME TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
-            EXPIRY_TIME TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
-            TIME_CREATED TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
-            POLL_TIME BIGINT,
-            STATUS VARCHAR (25) DEFAULT 'PENDING',
-            AUTHZ_USER VARCHAR (100),
-            TENANT_ID INTEGER,
-            USER_DOMAIN VARCHAR(50),
-            IDP_ID INTEGER,
-            PRIMARY KEY (DEVICE_CODE),
-            UNIQUE (CODE_ID),
-            UNIQUE (USER_CODE),
-            FOREIGN KEY (CONSUMER_KEY_ID) REFERENCES IDN_OAUTH_CONSUMER_APPS(ID) ON DELETE CASCADE
-)ENGINE INNODB;
+    CODE_ID VARCHAR(255),
+    DEVICE_CODE VARCHAR(255),
+    USER_CODE VARCHAR(25),
+    CONSUMER_KEY_ID INTEGER,
+    LAST_POLL_TIME TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    EXPIRY_TIME TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    TIME_CREATED TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    POLL_TIME BIGINT,
+    STATUS VARCHAR (25) DEFAULT 'PENDING',
+    AUTHZ_USER VARCHAR (100),
+    TENANT_ID INTEGER,
+    USER_DOMAIN VARCHAR(50),
+    IDP_ID INTEGER,
+    PRIMARY KEY (DEVICE_CODE),
+    UNIQUE (CODE_ID),
+    UNIQUE (USER_CODE),
+    FOREIGN KEY (CONSUMER_KEY_ID) REFERENCES IDN_OAUTH_CONSUMER_APPS(ID) ON DELETE CASCADE
+    )ENGINE INNODB;
 
 CREATE TABLE IF NOT EXISTS IDN_OAUTH2_DEVICE_FLOW_SCOPES (
-            ID INTEGER NOT NULL AUTO_INCREMENT,
-            SCOPE_ID VARCHAR(255),
-            SCOPE VARCHAR(255),
-            PRIMARY KEY (ID),
-            FOREIGN KEY (SCOPE_ID) REFERENCES IDN_OAUTH2_DEVICE_FLOW(CODE_ID) ON DELETE CASCADE
-)ENGINE INNODB;
+                                                             ID INTEGER NOT NULL AUTO_INCREMENT,
```

Comment:
Seems to be formatting issue. Please check

Reply:
Ack, @Chathuratharushka please note, as these changes are on top of https://github.com/wso2/docker-open-banking/pull/154 PR

Reply:
Fixed it by https://github.com/wso2/docker-open-banking/pull/154/commits/335f4a89ef0c9c7ecf0e5d74bbea5523fef7c643